presto-hive/src/main/java/io/prestosql/plugin/hive/authentication/HiveIdentity.java

```diff
@@ -25,6 +26,12 @@
 {
     private final String username;
 
+    // TODO: This should be called only by CachingHiveMetastore
```

Comment:
Maybe we should make this private and create a factory method `HiveIdentity.none()` for `CachingHiveMetastore` to use. That might make it more obvious what is happening.